modules/cms/controllers/ThemeOptions.php

```diff
@@ -1,8 +1,11 @@
 <?php namespace Cms\Controllers;
 
+use File;
+use Lang;
 use Backend;
 use BackendMenu;
 use ApplicationException;
+use Cms\Classes\Theme;
```

Comment:
These extra import statements shouldn't be required anymore

Reply:
Oops, indeed